packages/pwa/dom.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var hopsConfig = require('hops-config');
+
+module.exports = function installServiceWorker() {
+  return new Promise(function(resolve, reject) {
+    if (!('serviceWorker' in navigator)) {
+      return reject(
+        new Error('ServiceWorkers are not supported in this browser')
+      );
+    }
+
+    if (
+      window.location.protocol === 'https' ||
+      window.location.hostname === 'localhost'
+    ) {
+      window.addEventListener('load', function() {
+        resolve(
+          navigator.serviceWorker.register(hopsConfig.workerPath, {
+            scope: hopsConfig.basePath,
```

Comment:
Should we always use the `basePath` as scope or are there cases, where we would like to specify a scope that is different from the `basePath`?

Reply:
As `basePath` is the actual application's root path, I think we do not have to cater for this.